v2/internal/severity/severity.go

```diff
@@ -57,6 +58,16 @@ type SeverityHolder struct {
 	Severity Severity
 }
 
+func (severityHolder SeverityHolder) JSONSchemaType() *jsonschema.Type {
+	gotType := &jsonschema.Type{
+		Type:        "string",
+		Title:       "severity of the template",
+		Description: "Seriousness of the implications of the template",
+		Enum:        []interface{}{Info.String(), Low.String(), Medium.String(), High.String(), Critical.String()},
```

Comment:
Instead of manually listing the available enum values, please use the `severity.GetSupportedSeverities()` method. Based on the returned values you can dynamically create a `[]interface` with all the values

Reply:
Done